spec/lib/salus/scanners/report_go_dep_spec.rb

```diff
@@ -1,83 +1,64 @@
 require_relative '../../../spec_helper.rb'
 
 describe Salus::Scanners::ReportGoDep do
-  let(:blank_config) { {} }
-  let(:report) { Salus::Report.new }
-  let(:scan_report) { json_report['scans']['ReportGoDep'] }
-
   describe '#run' do
-    context 'no Gopkg.lock present' do
-      it 'should throw an error since there is nothing to parse' do
-        scanner = Salus::Scanners::ReportGoDep.new(
-          repository: Salus::Repo.new('spec/fixtures/blank_repository'),
-          report: report,
-          config: blank_config
-        )
-        expect { scanner.run }.to raise_error(
-          Salus::Scanners::Base::InvalidScannerInvocationError,
-          'Cannot report on Go dependencies without a Gopkg.lock file.'
-        )
-      end
+    it 'should throw an error if no Gopkg.lock is present' do
+      repo = Salus::Repo.new('spec/fixtures/blank_repository')
+      scanner = Salus::Scanners::ReportGoDep.new(repository: repo, config: {})
+
+      expect { scanner.run }.to raise_error(
+        Salus::Scanners::Base::InvalidScannerInvocationError,
+        'Cannot report on Go dependencies without a Gopkg.lock file.'
+      )
     end
 
-    context 'Gopkg.lock present' do
-      it 'should report on all the dependencies in the Gopkg.lock file' do
-        scanner = Salus::Scanners::ReportGoDep.new(
-          repository: Salus::Repo.new('spec/fixtures/report_go_dep'),
-          report: report,
-          config: blank_config
-        )
-        scanner.run
-        expect(scan_report['info']['dependency']).to include(
-          'dependency_file' => 'Gopkg.lock',
-          'type' => 'go_dep_lock',
-          'name' => 'github.com/PagerDuty/go-pagerduty',
-          'reference' => 'fe74e407c23e030fa1523e7cbd972398fd85ec5d',
-          'version_tag' => nil
-        )
-        expect(scan_report['info']['dependency']).to include(
-          'dependency_file' => 'Gopkg.lock',
-          'type' => 'go_dep_lock',
-          'name' => 'github.com/Sirupsen/logrus',
-          'reference' => 'ba1b36c82c5e05c4f912a88eab0dcd91a171688f',
-          'version_tag' => 'v0.11.5'
-        )
-        expect(scan_report['info']['dependency']).to include(
-          'dependency_file' => 'Gopkg.lock',
-          'type' => 'go_dep_lock',
-          'name' => 'golang.org/x/sys',
-          'reference' => '9a7256cb28ed514b4e1e5f68959914c4c28a92e0',
-          'version_tag' => nil
-        )
-      end
+    it 'should report on all the dependencies in the Gopkg.lock file' do
+      repo = Salus::Repo.new('spec/fixtures/report_go_dep')
+      scanner = Salus::Scanners::ReportGoDep.new(repository: repo, config: {})
+
+      scanner.run
+
+      dependencies = scanner.report.to_h.fetch(:info).fetch(:dependencies)
+
+      expect(dependencies).to match_array([
+                                            {
```

Comment:
nit: rad indentation.
Would move everything to under the `expect` call

Reply:
oops i trusted `rubocop -a` a bit too much